lib/image-stream/mime.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const _ = require('lodash');
+const Bluebird = require('bluebird');
+const fileType = require('file-type');
+const mime = require('mime-types');
+const fs = require('fs');
+
+/**
+ * @summary The default image MIME type
+ * @type {String}
+ * @constant
+ */
+exports.DEFAULT_MIME_TYPE = 'application/octet-stream';
+
+/**
+ * @summary Get archive mime type
```

Comment:
Should `archive` here be replaced with `a file's` ?

EDIT: And perhaps the `@summary` should explain that this method _might_ involve actually opening and reading the file itself, i.e. it might not _just_ look at the filename.